src/redux/ducks/locs/actions.js

```diff
@@ -59,21 +57,16 @@ const removeLOC = (address, account, hideModal) => (dispatch) => {
     });
 };
 
-const handleNewLOC = (locModel, time) => (dispatch) => {
+const handleNewLOC = (locModel) => (dispatch) => {
     dispatch(createLOCAction(locModel));
-    dispatch(notify(new LOCNoticeModel({time, loc: locModel, action: ADDED})))
 };
 
-const handleRemoveLOC = (address, time) => (dispatch, getState) => {
-    const loc = getState().get('locs').get(address);
+const handleRemoveLOC = (address) => (dispatch) => {
     dispatch(removeLOCAction({address}));
-    dispatch(notify(new LOCNoticeModel({time, loc, action: REMOVED})))
 };
 
-const handleUpdateLOCValue = (address, valueName, value, time) => (dispatch, getState) => {
-    const loc = getState().get('locs').get(address);
+const handleUpdateLOCValue = (address, valueName, value, time) => (dispatch) => {
     dispatch(updateLOCAction({valueName, value, address}));
-    dispatch(notify(new LOCNoticeModel({time, loc, action: UPDATED, params: {valueName, value} })))
 };
 
 const getLOCs = (account) => (dispatch) => {
```

Comment:
And please rework redux file structure/names for your modules as we discussed in slack.